.github/actions/getMergeCommitForPullRequest/action.yml

```diff
@@ -0,0 +1,15 @@
+name: 'Get merge commit for a pull request'
+description: 'Get the merge_commit_sha for a pull request'
+inputs:
+  GITHUB_TOKEN:
```

Comment:
Is this used?

Reply:
Ah, yes it is. This is a pretty significant change you missed while you were gone...

Basically the `markPullRequestsAsDeployed` action was encountering GitHub rate-limiting issues because we were trying to quickly comment on many PRs in quick succession. To fix that, we [started throttling our use of the GH api](https://github.com/Expensify/Expensify.cash/pull/2769). It turned out that the changes were pretty generic, and there was no case where we specific _don't_ want throttling, so we decided to implement the throttling in the GithubUtils class itself. So now it no longer takes an `octokit` instance in its constructor, but instead [maintains a singleton octokit instance which it initializes itself](https://github.com/Expensify/Expensify.cash/blob/ed2f64bb20abc9ae37faeffc9ffca0bf53a62e57/.github/libs/GithubUtils.js#L20-L55). And now _all_ the `GithubUtils`  functions are static. However, [it still grabs the token using the actions/core library](https://github.com/Expensify/Expensify.cash/blob/ed2f64bb20abc9ae37faeffc9ffca0bf53a62e57/.github/libs/GithubUtils.js#L32), so all the actions that use it still need this input.